config/initializers/admin_menu_item.rb

```diff
@@ -0,0 +1,8 @@
+Spree::Backend::Config.configure do |config|
+  config.menu_items << config.class::MenuItem.new(
+    [:braintree],
+    'wrench',
+    url: '/solidus_paypal_braintree/configurations/list',
+    condition: -> { can?(:admin, Spree::PaymentMethod) }
+  )
+end
```

Comment:
This configuration should probably live in [`engine.rb`](https://github.com/solidusio/solidus_paypal_braintree/blob/master/lib/solidus_paypal_braintree/engine.rb) and should really only be loaded if `SolidusSupport.backend_available?` is true.